bin/plugin/commands/performance.js

```diff
@@ -162,80 +211,87 @@ async function runPerformanceTests( branches, options ) {
+		//     5.7   -> 5.7   (unchanged)
+		//     5.7.0 -> 5.7   (changed)
+		//     5.7.2 -> 5.7.2 (unchanged)
+		const zipVersion = options.wpVersion.replace( /^(\d+\.\d+).0/, '$1' );
+		wpZipUrl = `https://wordpress.org/wordpress-${ zipVersion }.zip`;
+	}
 
-	log( '\n>> Preparing an environment directory per branch' );
-	const branchDirectories = {};
+	const branchDirs = {};
 	for ( const branch of branches ) {
-		log( `    >> Branch: ${ branch }` );
-		const sanitizedBranch = sanitizeBranchName( branch );
-		const environmentDirectory = rootDirectory + '/envs/' + sanitizedBranch;
+		logStep( 2, 'Branch:', fAccent( branch ) );
+		const sanitizedBranchName = sanitizeBranchName( branch );
+		const envDir = path.join( envsDir, sanitizedBranchName );
+
+		logStep( 3, 'Creating directory:', fAccent( envDir ) );
+		fs.mkdirSync( envDir );
 		// @ts-ignore
-		branchDirectories[ branch ] = environmentDirectory;
-		const buildPath = `${ environmentDirectory }/plugin`;
-		await runShellScript( 'mkdir ' + environmentDirectory );
-		await runShellScript( `cp -R ${ baseDirectory } ${ buildPath }` );
+		branchDirs[ branch ] = envDir;
+		const buildDir = path.join( envDir, 'plugin' );
 
-		const fancyBranch = formats.success( branch );
+		logStep( 3, 'Copying source to:', fAccent( buildDir ) );
+		await runShellScript( `cp -R ${ sourceDir } ${ buildDir }` );
 
-		if ( branch === options.testsBranch ) {
-			log(
-				`        >> Re-using the testing branch for ${ fancyBranch }`
-			);
-			await runShellScript(
-				`cp -R ${ performanceTestDirectory } ${ buildPath }`
-			);
-		} else {
-			log( `        >> Fetching the ${ fancyBranch } branch` );
-			// @ts-ignore
-			await SimpleGit( buildPath ).reset( 'hard' ).checkout( branch );
-		}
+		logStep( 3, 'Checking out:', fAccent( branch ) );
+		// @ts-ignore
+		await SimpleGit( buildDir ).raw( 'checkout', branch );
 
-		log( `        >> Building the ${ fancyBranch } branch` );
+		logStep( 3, 'Installing dependencies and building' );
 		await runShellScript(
-			'bash -c "source $HOME/.nvm/nvm.sh && nvm install && npm ci && npm run prebuild:packages && node ./bin/packages/build.js && npx wp-scripts build"',
-			buildPath
+			`bash -c "${ [
+				'source $HOME/.nvm/nvm.sh',
+				'nvm install',
+				'npm ci',
+				'npm run build',
+			].join( ' && ' ) }"`,
+			buildDir
 		);
 
-		// Create the config file for the current env.
+		const wpEnvConfigPath = path.join( envDir, '.wp-env.json' );
+
+		logStep( 3, 'Saving wp-env config to:', fAccent( wpEnvConfigPath ) );
+
 		fs.writeFileSync(
-			path.join( environmentDirectory, '.wp-env.json' ),
+			wpEnvConfigPath,
 			JSON.stringify(
 				{
 					config: {
 						WP_DEBUG: false,
 						SCRIPT_DEBUG: false,
 					},
-					core: 'WordPress/WordPress',
-					plugins: [ path.join( environmentDirectory, 'plugin' ) ],
-					themes: [
-						path.join(
-							performanceTestDirectory,
-							'test/emptytheme'
-						),
-					],
+					core: wpZipUrl || 'WordPress/WordPress',
```

Comment:
is this a behavioral change here?

Reply:
Nope! Previously, the `wpZipUrl` (if set) was being added to the config **after** the config was created and saved, so it was an unnecessary step where we needed to reopen, alter, and save the config again.